agents/aliyun/fence_aliyun.py

```diff
@@ -1,52 +1,66 @@
 #!@PYTHON@ -tt
 
-import sys, re
+import sys
 import logging
 import atexit
 import json
+
 sys.path.append("@FENCEAGENTSLIBDIR@")
 from fencing import *
-from fencing import fail, fail_usage, EC_TIMED_OUT, run_delay
+from fencing import fail_usage, run_delay
+
 
 try:
 	from aliyunsdkcore import client
 	from aliyunsdkcore.auth.credentials import EcsRamRoleCredential
+	from aliyunsdkcore.profile import region_provider
+except ImportError as e:
+	fail_usage("The 'aliyunsdkcore' module has been not installed or is unavailable, try to execute the command 'pip install aliyun-python-sdk-core --upgrade' to solve. error: %s" % e)
+
+
+try:
 	from aliyunsdkecs.request.v20140526.DescribeInstancesRequest import DescribeInstancesRequest
 	from aliyunsdkecs.request.v20140526.StartInstanceRequest import StartInstanceRequest
 	from aliyunsdkecs.request.v20140526.StopInstanceRequest import StopInstanceRequest
 	from aliyunsdkecs.request.v20140526.RebootInstanceRequest import RebootInstanceRequest
-	from aliyunsdkcore.profile import region_provider
-except ImportError:
-	pass
+except ImportError as e:
+	fail_usage("The 'aliyunsdkecs' module has been not installed or is unavailable, try to execute the command 'pip install aliyun-python-sdk-ecs --upgrade' to solve. error: %s" % e)
```

Comment:
It should pass here to avoid having to install all the libraries when doing builds.

Reply:
Okay, but can I use logging.error to output this message to remind users?

Reply:
or .warn().